chapter/design-principles.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Frontend Uebung</title>
+    <link rel="stylesheet" href="../CSS/stylesheet.css">
+
+</head>
+
+
+<body>
+
+    <!-- Erste Section -->
+    <section class="firstSection">
+        <div class="divIntro">
+            <h1>Design Principles</h1>
+            <p class="intro">
+                The challenge is to develop an application that tries to increase the user’s participation in team 
+                sports through online communities. Several factors play into the success of this concept. It is not 
+                possible to achieve this goal if the application is not properly designed. To achieve this, design 
+                principles of several disciplines are applied to achieve a user-centric solution.
+            </p>
+
+            <nav>
+                <a href="http://127.0.0.1:5500/index.html">Inhaltsverzeichnis</a>
+            </nav>
+        </div>   
+    </section>     
+
+    <!-- Zweite Section -->
+    <section>
+        <div class="divBase">
+            <h2>User Experience Design</h2>
+                <p>
+                    The term "user experience" was first coined by Don Norman in the early 1990s when he was working at
+                    Apple (NORMAN, 2013, P. XIII F.). User experience comprises all aspects of a user’s interaction with
+                    a company, its services, and its products. The focus and most important goal of good user experience
+                    is to meet the user’s needs in every aspects. Close cooperation is required between different disciplines,
+                    such as engineering, marketing, graphic design, industrial design, and interface design (NORMAN & NIELSEN, N.D.).
+                </p>
+        </div>
+
+        <div class="divQuote">
+            <blockquote cite="https://en.wikipedia.org/wiki/The_Design_of_Everyday_Things">
+                <p>
+                    »Good design is actually a lot harder to notice than poor design, in part because good designs fit our needs so
+                    well that the design is invisible, serving us without drawing attention to itself. Bad design, on the other hand,
+                    screams out its inadequacies, making itself very noticeable.«
+                </p>
+                <footer>
+                    <cite>
+                        <a href="https://jnd.org">Don Norman</a>
+                    </cite>
+                </footer>
+            </blockquote>
+        </div>
+
+        <div class="divBase">
+            <p>
+                User experience captures everything a user experiences while using a given product, from the visuals, the haptics,
+                and the audio to how the product feels. User experience of a product comprises three primary factors: its look,
+                feel, and usability (IXDF - UX DESIGN, N.D.) (→ FIG. 04).
+            </p>
+        </div>
```

Comment:
Jeden Paragraphen in ein `<div>`packen ist überflüssig und macht das Dokument unnötig komplex. Wenn du dies für Styles verwenden möchtest, würde ich empfehlen, die `<p>`s mit einer Klasse zu versehen oder das umschließende Element (`<section>`) zu stylen.